core-bundle/src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -586,14 +586,14 @@ function (array $matches) use ($arrData)
 
 		$evaluateExpression = function ($strExpression) use ($arrData)
 		{
-			if (!preg_match('/^([^=!<>\s]+)([=!<>]+)(.+)$/s', $strExpression, $arrMatches))
+			if (!preg_match('/^([^=!<>\s]+)[ ]*([=!<>]+)(.+)$/s', $strExpression, $arrMatches))
 			{
 				return false;
 			}
 
 			$strToken = $arrMatches[1];
 			$strOperator = $arrMatches[2];
-			$strValue = $arrMatches[3];
+			$strValue = trim($arrMatches[3], " \t");
```

Comment:
```suggestion
			$strValue = trim($arrMatches[3], " ");
```

Reply:
With single quotes though:

```suggestion
			$strValue = trim($arrMatches[3], ' ');
```